### DISCUSSION TITLE
Making tiles out of polygons

### DISCUSSION THREAD

**Discussion by sumew** [maplibre/maplibre-gl-js#2347] [Q&A]
I want to use maplibre-gl-js to display a detailed admin map in an environment where the network bandwidth is very low. I have started with a geojson polygon of detailed admin maps and have converted to pmtiles with the intention of hosting it in cloudflare. The problem is I can't seem to get it to display. I used this example and replaced the provided pmtiles with mine
https://github.com/protomaps/PMTiles/blob/main/js/examples/maplibre.html

It looks like one cannot simply create tiles out of polygongs and use it as a source but I don't know what else is missing from my tiles. What is the minimum set of things I need to project my polygons on an empty world map?

Thanks for all the help!

**Comment by HarelM** [comment]
The content of the pmtiles should be mvt as far as I know.
You could also play around with `addProtocol` I guess to get things done.
I'm not an expert on pmtiles, you could open a discussion in the pmtiles repo for assistance.
